gunicorn.conf.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+
+from gunicorn.arbiter import Arbiter
```

Comment:
We use gunicorn here and it will not be recognized if we don't add it to dev requirements

Reply:
Is it really an issue? The CI doesn't complain, if we don't install and use Gunicorn this file won't be used.

Reply:
The CI doesn't complain because ruff still didn't implement E0401 rule.

I think we should give to developers every modules we use in our code.

Reply:
Actually do we need to import the Arbiter class ? I believe we do not need to type `on_starting` `server` argument

Reply:
We don't really, the objective was to type the method, but we don't use the `server` argument. I will just add a comment